docs/guides/theme_architecture/blocks/schema/variables/variable_types/color.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Color
+parent: Variable types
+---
+
+# Color
+
+Renders a colour picker, from which a Creator can choose a colour from the theme palette defined in [Site Styles]({% link docs/guides/site_styles/index.md %}), or a custom hex colour.
+Returns a [Color]({% link docs/reference/objects/color.md %}) object.
+
+##### syntax
+{% raw %}
+```
+---
+my_variable:
+    type: color
+    default:
+        palette: body-color
+---
+
+{{my_variable.rgba}}
+
```

Comment:
could note that you can use `my_variable.hex` to use an opaque color even if the Creator added transparency